chia/server/node_discovery.py

```diff
@@ -194,8 +195,6 @@ async def start_client_async(self, addr: PeerInfo, is_feeler: bool) -> None:
         try:
             if self.address_manager is None:
                 return
-            if addr.host in self.pending_outbound_connections:
```

Comment:
this check was moved to outside of the task. it seemed unnecessary to create a taks just to then exit because we already had a pending outgoing connection. See line 367